test/test.py

```diff
@@ -0,0 +1,98 @@
+import munge
+import unittest
+
+
+def file_factory(n):
+    # makes a file with "the great big whale splashed the timid blue fox.\n" repeated n times
```

Comment:
You should make these actual docstrings (outlined by three quotes)

Reply:
For functions you don't expect users to use, I'd expect something like

```
def file_factory(n): 
    """Writes a file with n lines of the sentence 'The great big whale splashed the timid blue fox' to the file location test_files/simple_whale_{n}.txt."""
```

For fancier functions, it should probably be more complex:

```
def file_factory(n, filename, sentence=DEFAULT_SENTENCE): 
    """Writes a file with n lines of DEFAULT_SENTENCE to the specified filename.
    Arguments:
        n (int): number of lines in the output file
        filename (str): string path to where the file should be written
        sentence (str): What to write in each line. By default, DEFAULT_STRING.

   Returns: none
    """
```